plan/logical_test.go

```diff
@@ -494,6 +494,33 @@ func TestLogicalPlanner(t *testing.T) {
 						Fn: interpreter.ResolvedFunction{
 							Scope: valuestest.Scope(),
 							Fn: &semantic.FunctionExpression{
+								Vectorized: &semantic.FunctionExpression{
```

Comment:
The changes in `plan/logical_test` are a little unexpected. I suppose an explanation is the flag just wasn't set for these tests prior to this change?

The planner rule was added after the vec repeat work was done, iirc.

Reply:
> The changes in `plan/logical_test` are a little unexpected. I suppose an explanation is the flag just wasn't set for these tests prior to this change?

That seems to be the case. I guess whoever added the tests just didn't account for the flag being on.

Reply:
Got it. Thanks for doing this!